integration-tests/bats/sql-local-remote.bats

```diff
@@ -439,3 +451,48 @@ get_staged_tables() {
     [ "$status" -eq 1 ]
     [[ "$output" =~ "When a password is provided, a user must also be provided" ]] || false
 }
+
+@test "sql-local-remote: verify dolt conflicts resolve behavior" {
+  skip "This test relies on dolt commit being migrated"
```

Comment:
Commit has already been migrated. You mean checkout?

More to the point, let's not skip tests in this file for this project. In the worst case situation, you can do `dolt sql -q "call dolt_checkout()"`. There should always be some way to do what you want without using the CLI commands

Reply:
Removing `skip` directive and rewriting in such a way that we don't need to use a migrated version of `dolt checkout`.